halo2-base/Cargo.toml

```diff
@@ -17,7 +17,12 @@ serde_json = "1.0"
 log = "0.4"
 
 # Use Axiom's custom halo2 monorepo for faster proving when feature = "halo2-axiom" is on
-halo2_proofs_axiom = { git = "https://github.com/axiom-crypto/halo2.git", branch = "axiom/dev", package = "halo2_proofs", optional = true }
+# halo2_proofs_axiom = { git = "https://github.com/axiom-crypto/halo2.git", branch = "axiom/dev", package = "halo2_proofs", optional = true }
```

Comment:
Can you change this back to the main branch of `axiom-crypto/halo2`?

Reply:
In `halo2-ecc/src/bn254/tests/bls_signature.rs line:59-60`, I'm using iterators for affine points (I've marked a comment there for your reference). The iterator is defined in halo2curves (though I've previously mistakenly lumped it in the `bls12-381/dev` PR/branch). I've created a new [PR](https://github.com/axiom-crypto/halo2curves/pull/2) there